### DISCUSSION TITLE
Set `Content-Id` header on attachments through send message API

### DISCUSSION THREAD

**Discussion by memcorrupt** [postalserver/postal#2817] [Feature suggestions]
Good to see that postalserver seems to have resumed development. 🙌 

Currently, there is no support for setting the `Content-Id` header on attachments sent through `/api/v1/send/message`. I'm not quite well versed enough with the project & programming in Ruby to pull request it at this time, but from my brief look into the source code, I believe it can be implemented pretty easily.

1. Add `cid` parameter and handling to attachments in [send_api_controller.rb](https://github.com/postalserver/postal/blob/main/api/controllers/send_api_controller.rb), so that it gets passed to `OutgoingMessagePrototype.new`
2. Set `content_id` key within mail.attachments[name].
    - https://github.com/postalserver/postal/blob/main/app/models/outgoing_message_prototype.rb#L177
    - https://github.com/mikel/mail/blob/master/lib/mail/attachments_list.rb#L84

If there is any documentation that could ease the setup of a development environment for Postal, I could attempt to make the changes and PR them upstream.

**Comment by olegbliaher** [comment]
That would be a nice feature.

**Reply by daryljones** [reply]
Is there a workaround for displaying inline images?   The ability to set Content-Id is needed.
